server/internal/core/application/utils.go

```diff
@@ -411,6 +426,37 @@ func (m *forfeitTxsMap) allSigned() bool {
 	return true
 }
 
+type redeemTxRequests struct {
```

Comment:
given https://github.com/ark-network/ark/pull/537/files#r2044873760, i'd rename this util struct to a generic `outpointMap` that can be reused for both tx request and redeem tx inputs.

Reply:
Done